src/core/qgslegendstyle.h

```diff
@@ -121,6 +121,22 @@ class CORE_EXPORT QgsLegendStyle
      */
     void setAlignment( Qt::Alignment alignment ) { mAlignment = alignment; }
 
+    /**
+    * Returns the indent of a group or subgroup.
+    *
```

Comment:
Can you add a note in these methods that the units are millimeters?

Reply:
Done